about.js

```diff
@@ -0,0 +1,52 @@
+const hamburgerone = document.getElementById('hamone');
+hamburgerone.addEventListener('click', () => {
+  document.getElementById('iconlistone').style.width = '100%';
+});
+
+const closeNavone = document.getElementById('closeNavone');
+closeNavone.addEventListener('click', () => {
```

Comment:
- [ ] Right now, your close button is broken, you should consider fixing it. This is how it looks right now:

![image](https://user-images.githubusercontent.com/66967274/143444163-ed92dbf6-dccd-43d7-98cf-3d8dbe58723c.png)